app/models/Document.ts

```diff
@@ -305,6 +305,8 @@ export default class Document extends BaseModel {
           },
           {
             lastRevision: options.lastRevision,
+            publish: options?.publish,
+            done: options?.done,
```

Comment:
admittedly, I'm not 100% sure what "done" does system wide, but this brings it back in line with the previous implementation

Reply:
`done` is almost a relic now, it was originally a signal to the backend that editing session is complete. It makes sense to include it here and remove everywhere later.